core/domain/user_jobs_one_off.py

```diff
@@ -135,6 +139,97 @@ def reduce(key, stringified_values):
                 subscription_services.subscribe_to_collection(key, item['id'])
 
 
+class UserImpactCalculationOneOffJob(jobs.BaseMapReduceJobManager):
+    """ Calculates each user's impact score, where impact score is defined as:
+    Sum of (
+    ln(playthroughs) * (ratings_scaler) * (average(ratings) - 2.5))
```

Comment:
Both of these are addressed in the actual implementation-- 0 is returned if average(ratings) - 2.5 < 0, and 0 is also returned if the number of playthroughs is not > 0. I will change this description to account for these cases-- do you think max(what it says now, 0) should suffice to clarify?